server/controllers/userPostController.js

```diff
@@ -0,0 +1,127 @@
+const jwt = require('jsonwebtoken');
+const mongoose = require('mongoose');
+const PostModel = require('../models/postSchema');
+const config = require('../config');
+const _ = require('lodash');
+
+const User = mongoose.model('User');
+
+exports.me = (req, res) => {
+  if (!req.headers.authorization) {
+    return res.status(401).end();
+  }
+
+  const token = req.headers.authorization.split(' ')[1];
+
+  return jwt.verify(token, config.jwtSecret, (err, decoded) => {
+    if (err) {
+      return res.status(401).end();
+    }
+    const userId = decoded.sub;
+    return res.status(200).send({ userId });
```

Comment:
Why do you need this?

Reply:
I think there is no need of this.

Reply:
Yes this was from before @in42 merge, so I did not have any `user` things to work with, so used this. But now I have changed and refactored to use from @in42 code. I will remove this in the next commit.